### DISCUSSION TITLE
List order

### DISCUSSION THREAD

**Discussion by jano23** [Natizyskunk/vscode-sftp#340] [Q&A] [answered]
When I invoke **SFTP: Upload Changed Files**, SFTP presents me with a list of repositories to choose from.
I would love to have the ability to change the order of the list that shows up.
The list is not in alphabetical order

**Comment by Natizyskunk** [comment, marked as answer]
@jano23 You can open a new issue for this respecting the template or if you now how to add it to the code, you can open a pull request.
I'll take a look at ASAP.
